saga/seata-saga-engine-store/src/main/java/io/seata/saga/engine/store/db/DbAndReportTcStateLogStore.java

```diff
@@ -68,25 +68,25 @@
  */
 public class DbAndReportTcStateLogStore extends AbstractStore implements StateLogStore {
 
-    private static final Logger                                   LOGGER                       = LoggerFactory.getLogger(
```

Comment:
Please restore the unchanged lines of code, otherwise it will affect the efficiency of the review